PowerUp/app/src/main/java/powerup/systers/com/IntroActivity.java

```diff
@@ -0,0 +1,63 @@
+package powerup.systers.com;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.graphics.Color;
+import android.os.Bundle;
+
+import com.github.paolorotolo.appintro.AppIntro;
+import com.github.paolorotolo.appintro.AppIntroFragment;
+
+public class IntroActivity extends AppIntro {
+
+    private SharedPreferences settings;
+    private boolean firstRun;
+    @Override
+    public void init(Bundle savedInstanceState) {
+
+        settings = getSharedPreferences("prefs", 0);
+        firstRun = settings.getBoolean("firstRun", true);
+        if (firstRun) {
+
+            addSlide(new IntroFragment1());
+            addSlide(new IntroFragment2());
+            addSlide(new IntroFragment3());
+            addSlide(new IntroFragment4());
+            addSlide(new IntroFragment5());
+            addSlide(new IntroFragment6());
+        }
+        else{ loadMainActivity(); }
+    }
+
+    private void loadMainActivity(){
```

Comment:
Add a space before `{`. Again, if you run `Code->Reformat Code`, it should get fixed